articles/openshift/oshift-sco.md

```diff
@@ -114,90 +105,172 @@ Containers in OpenShift by default don't persist data. Every time an application
 
-As your workloads increase or become more critical and you want to increase the performance and resilience of your Elasticsearch cluster by scaling it across three nodes, you can deploy an additional Infrastructure Node (see below for [Infrastructure Node](#infrastructure-nodes) costs).
+#### Enhanced metric collection
 
-### Enhanced metric collection
+UKCloud will provision a Prometheus-based metrics platform to collect and publish your cluster's metrics from all containers and components in one user interface. This will run on infrastructure nodes, if you have them, or application nodes otherwise.
 
-As part of your Foundation Pack, UKCloud will provision a single Hawkular metrics instance backed by a single Cassandra instance to collect and publish your cluster's metrics from all containers and components in one user interface.
+To provide this service, UKCloud will need to provision an initial allocation of Tier 2 persistent volume, charged at the prevailing rate, to store this data.
 
-To provide this service, UKCloud will need to provision an initial 10GiB of Tier 2 Persistent Volume at the prevailing rate.
+To provide resilience and added performance to your cluster's metrics, you can deploy additional infrastructure nodes.
 
-To provide resilience and added performance to your cluster's metrics, you can deploy additional Infrastructure Nodes (see below for [Infrastructure Node](#infrastructure-nodes) costs).
+#### SSL certificates
 
-### Infrastructure nodes
+By default, UKCloud will provide 90-day Let's Encrypt SSL certificates for your OpenShift cluster. At the point of deployment, you can provide your own certificates for use with OpenShift.
 
-Additional nodes to support OpenShift core services such as OpenShift routers, registry, metric collection and aggregated logging. Each Foundation Pack is deployed with three Infrastructure Nodes. Where required, you can deploy additional Infrastructure Nodes for a flat monthly fee, with each Infrastructure Node providing an additional 2vCPUs and 16GiB RAM for OpenShift supporting services. You cannot use Infrastructure Nodes for containerised workloads.
+#### Container native storage
 
-### SSL certificates
+Kubernetes was not designed with stateful applications like databases in mind. For enterprise applications, these systems have non-negotiable business requirements like high availability, data security, disaster recovery, strict performance SLAs and hybrid/multi-cloud operations.
 
-By default, UKCloud will provide 90 day Let's Encrypt SSL certificates for your OpenShift cluster. At the point of deployment, you can to provide your own certificates for use with OpenShift.
+To address the challenge above, UKCloud offers a [Portworx Enterprise](../openshift/oshift-sd-portworx.md) service, assisting with the adoption of Kubernetes for mission critical enterprise and traditional/legacy applications.
```

Comment:
I'd remove this line entirely, I think portworx is being retired?

Reply:
If it's not retired we could talk about it for DR and for an alternative approach to persistent storage - but I'd suggest we elaborate some more on why.

Reply:
Removed for now